ideascube/tests/test_settings.py

```diff
@@ -8,7 +8,7 @@ def test_all_settings(request):
     assert confdir.check(dir=True)
 
     for path in confdir.listdir():
-        if path.ext != 'py':
+        if path.ext != '.py':
```

Comment:
Wait what?

I'm absolutely sure I got errors when I voluntarily introduced bad things in the config files, at the time I wrote this test. :confused: 

Could it have been a change in `py`?